packages/mobile/src/web3/testnets.ts

```diff
@@ -11,4 +11,4 @@ export enum Testnets {
 }
 
 export const DEFAULT_TESTNET: Testnets = Config.DEFAULT_TESTNET
-export const DEFAULT_SYNC_MODE: GethSyncMode = parseInt(Config.DEFAULT_SYNC_MODE, 10)
+export const DEFAULT_SYNC_MODE: GethSyncMode = _DEFAULT_SYNC_MODE
```

Comment:
Sorry I should have been more clear with my feedback. I don't think this needs to be in this file anymore.
Makes no sense to export it from 2 places :)